src/Microsoft.AspNetCore.SpaServices/Webpack/ConditionalProxyMiddleware.cs

```diff
@@ -9,7 +9,7 @@
 namespace Microsoft.AspNetCore.SpaServices.Webpack
 {
     /// <summary>
-    /// Based on https://github.com/aspnet/Proxy/blob/dev/src/Microsoft.AspNetCore.Proxy/ProxyMiddleware.cs
+    /// Based on https://github.com/aspnet/Proxy/blob/master/src/Microsoft.AspNetCore.Proxy/ProxyMiddleware.cs
```

Comment:
Might be best to shorten this to just `Based on ProxyMiddleware from https://github.com/aspnet/Proxy/.` that way we don't have to worry about adjusting this in the future.